src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeRevurderFraValidering.kt

```diff
@@ -48,99 +41,67 @@ object VilkårperiodeRevurderFraValidering {
         }
     }
 
-    fun validerEndrePeriodeRevurdering(
-        behandling: Saksbehandling,
+    /**
+     * Valider at det ikke er gjort noen endringer på vilkårperiode
+     * Antar at resultatet ikke er endret dersom ingen svar er endret
+     */
+    fun validerAtKunTomErEndret(
         eksisterendePeriode: Vilkårperiode,
-        oppdatertPeriode: Vilkårperiode,
+        oppdatertPeriode: LagreVilkårperiode,
+        revurderFra: LocalDate,
     ) {
-        if (behandling.type != BehandlingType.REVURDERING) {
-            return
-        }
-
-        val revurderFra = behandling.revurderFra
-
-        validerRevurderFraErSatt(revurderFra)
-
-        if (eksisterendePeriode.fom >= revurderFra) {
-            feilHvis(oppdatertPeriode.fom < revurderFra) {
-                "Kan ikke sette fom før revurder-fra(${revurderFra.norskFormat()})"
-            }
-            return
-        }
-
         feilHvis(eksisterendePeriode.fom != oppdatertPeriode.fom) {
             "Kan ikke endre fom til ${oppdatertPeriode.fom.norskFormat()} fordi " +
                 "revurder fra(${revurderFra.norskFormat()}) er før. Kontakt utviklingsteamet"
         }
         feilHvis(oppdatertPeriode.tom < revurderFra.minusDays(1)) {
             "Kan ikke sette tom tidligere enn dagen før revurder-fra(${revurderFra.norskFormat()})"
         }
-        feilHvis(eksisterendePeriode.faktaOgVurdering.type != oppdatertPeriode.faktaOgVurdering.type) {
-            logEndring(eksisterendePeriode, oppdatertPeriode)
-            "Kan ikke endre type på perioden. Kontakt utviklingsteamet"
-        }
-        feilHvis(eksisterendePeriode.faktaOgVurdering.fakta != oppdatertPeriode.faktaOgVurdering.fakta) {
+        feilHvis(eksisterendePeriode.faktaOgVurdering.tilFaktaOgSvarDto() != oppdatertPeriode.faktaOgSvar) {
```

Comment:
Hvorfor gjør om eksistrendePeriode til FaktaOgSvarDto? Virker litt kunstig. Er det noen måter å sammenlikne faktaOgVurdering med faktaOgSvar?

Reply:
Det er for å sammenligne svar mot svar. Hadde vært mer naturlig å snu det motsatt vei, men det krever å gjøre om svarene til vurdering, altså svar + resultat. Og da er jeg tilbake til å måtte sende med flagg eller lage en annen funksjon for å mappe om enn den som finnes